onmidimessage event change (#13664)
mdn/content: files/en-us/web/api/midiinput/index.md

_This interface doesn't implement any specific properties, but inherits properties from {{domxref("MIDIPort")}}._

> ### Events

Reviewer: This section must be after the _Methods_ section, according to the template.

> - {{domxref("MIDIInput.midimessage_event")}}

Reviewer: ```suggestion
- {{domxref("MIDIInput.midimessage_event", "midimessage"}}
```

>   - : When the current port receives a MIDI message it triggers a `midimessage` event.

Reviewer: ```suggestion
  - : Fired when the current port receives a MIDI message.
```

### Events

> - {{domxref("MIDIInput.midimessage_event", "midimessage"}}

Reviewer: This is the test failure.

```suggestion
- {{domxref("MIDIInput.midimessage_event", "midimessage")}}
```

Author: Ooo good catch!
